backend/app/repositories/base.py

```
@@ -0,0 +1,69 @@
+    """Base repository class for CRUD operations."""
+
+    def __init__(self, model: Type[ModelType]):
+        """Initialize repository with model."""
+        self.model = model
+
+    def get(self, db: Session, id: Any) -> Optional[ModelType]:
+        """Get a single record by ID."""
+        return db.get(self.model, id)
+
+    def get_multi(
+        self, db: Session, *, skip: int = 0, limit: int = 100
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)